web-client/src/modules/requests/containers/NewRequestsContainer/NewRequestsContainer.tsx

```diff
@@ -29,6 +29,12 @@ const RequestDetails = styled.div`
   background: white;
 `;
 
+const MapContainer = styled.div`
```

Comment:
@btuerker It would be nicer if we could use a media query rather than giving it a specific height
https://www.w3schools.com/cssref/css3_pr_mediaquery.asp

I had played around with the proper height of the map.  The problem is that the NewRequest form takes up so much space on smaller devices that not enough is left for the map -- at least on the screens provided by my google chrome responsive mode.   

The css in the NewRequestForm needs to be tighter.  It takes up too much height.

We solved this by wrapping the NewRequestForm and the Map in a flex container.

I had had performance issues with heights < 100%:  Either the center marker would disappear below the UI or gray bars would appear around the map when the user increased or decreased the zoom of the browser itself -- the map wouldn't fill the entirety of its HTML box.

That said, I am okay for now giving this component a height less than 100% to resolve the urgent bug, but we want to test for any bugs this might create, and we may want to wrap this in a media query for different sized screens

Reply:
> @sabind I would like your opinion on this.
> This fixes the issue on mobile displays and now it shows up perfectly fine on mobile.
> But it doesn't show the request form anywhere in the desktop view, not even when we scroll the page
> ![image](https://user-images.githubusercontent.com/13694998/84334789-de46d800-abb0-11ea-8870-862a3d77d70c.png)
> 
> I would like to request @btuerker for fixing this, but should we accept this now and fix it later? because mobile is our top priority right? so we could deploy this now and @btuerker could work on fixing this issue on the desktop by tomorrow or he could fix it and we will merge it tomorrow along with the fix

max-height would solve this, for now, I'm considering that we'll entirely change this screen on desktop view. Figma designs show that desktop view uses modal for request create operation. Also, I can put a max-width to the parent component and aligned it to the center. I say that again, just for this moment. Here's how it would look:

![Screen Shot 2020-06-11 at 6 39 15 PM](https://user-images.githubusercontent.com/46192266/84407015-f51e1680-ac12-11ea-841f-72be1e051d5f.png)

Reply:
I think adding @media queries would help to solve this problem

Reply:
@mcverter @btuerker It' sounds like we're making some progress here. Let's see if we can't improve this, but I like the progress we've made. I don't honestly know if mobile or desktop will be more popular. I think it depends on age. For example PINs may be more desktop based (or at least iPad).

Reply:
@sabind @btuerker just add a media query on line 33 to adjust

Reply:
@sabind I'm planning to improve the desktop layout for the later versions. At least, we should remove the bottom navbar and show the sider menu on left.

However, @sabind can I just fix the desktop view like below: 
![Screen Shot 2020-06-11 at 7 19 49 PM](https://user-images.githubusercontent.com/46192266/84412899-9360ab00-ac18-11ea-9f9f-9cd74ee5c82a.png)